tests/run_tests.sh

```diff
@@ -285,7 +306,7 @@ function run_group_tests()
 function run_individual_tests()
 {
     EXIT_CODE=0
-
+    USE_PY3=0
```

Comment:
I think initialization of this variable should be inside the for loop and just before checking file `python3_test_files.txt`.